cli/errors.rs

```diff
@@ -112,17 +111,5 @@ pub fn get_error_class_name(e: &AnyError) -> &'static str {
       e.downcast_ref::<ResolutionError>()
         .map(get_resolution_error_class)
     })
-    .unwrap_or_else(|| {
-      if cfg!(debug) {
-        log::warn!(
-          "Error '{}' contains boxed error of unknown type:{}",
-          e,
-          e.chain().fold(String::new(), |mut output, e| {
-            let _ = write!(output, "\n  {e:?}");
-            output
-          })
-        );
-      }
-      "Error"
-    })
```

Comment:
Why was this removed?

Reply:
I suggested to. It doesn't seem useful?

Reply:
See https://github.com/denoland/deno/pull/24721

Reply:
Okay, that makes sense